k8s/claim/async_deprovision.go

```diff
@@ -64,5 +65,6 @@ func pollDeprovisionState(
 			return mode.LastOperationStateFailed
 		}
 		pollState = newState
+		time.Sleep(time.Duration(30) * time.Second)
```

Comment:
worth making this time configurable? also, you don't need the `time.Duration(...)` around the `30`